.github/workflows/grapl-release.yml

```diff
@@ -11,6 +11,37 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Configure git
+        run: |
+          git config --global user.name 'Grapl CI'
+          git config --global user.email 'grapl.code@graplsecurity.com'
+
+      - name: Install bump2version CLI
+        run: |
+          python -mvenv venv && . venv/bin/activate
+          pip install bump2version
+          deactivate
+
+      - name: Bump grapl_graph_descriptions version
+        run: |
+          . venv/bin/activate
+          cd src/rust/graph-descriptions
+          bump2version --no-tag patch
```

Comment:
One weird consequence here is that we'll never publish a `.0` patch version to PyPI. We'll never publish `grapl_graph_descriptions 1.0.0` or `2.0.0`. Instead, the version will be e.g. `2.0.1`. OTOH, we'll never fail to publish due to extant versions, which is the bug this PR is meant to address.

Reply:
Weird, but whatever.